index.js

```diff
@@ -0,0 +1,113 @@
+import { darkTheme, lightTheme } from "./theme.js"
+import { createNewSideNote, generateId, shortenContent } from "./utils.js"
+
+let storedNotes = JSON.parse(localStorage.getItem("notes"))
+let activeId = ""
+
+const toggle = document.querySelector('input[type="checkbox"]')
+const sidebar = document.querySelector(".sidebar")
+const newButton = document.querySelector("#btn-new")
+const headingDiv = document.querySelector("#heading")
+const contentDiv = document.querySelector("#content")
+init()
+const sidebarNotes = document.querySelectorAll(".sidebar-note")
+
+// set active class to new note and remove from prev note in sidebar
+function makeNoteActive(id) {
+  if (activeId !== id) {
+    {
+      const sidebarNote = document.querySelector(`#${id}`)
+      const sidebarHeading = document.querySelector(`#${id} > h3`)
+      sidebarNote.classList.add("sidebar-note-active")
+      sidebarHeading.classList.add("sidebar-note-heading-active")
+      if (activeId) {
+        const oldNote = document.querySelector(`#${activeId}`)
+        const oldHeading = document.querySelector(`#${activeId} > h3`)
+        oldNote.classList.remove("sidebar-note-active")
+        oldHeading.classList.remove("sidebar-note-heading-active")
+      }
+      activeId = id
+      const { heading, content } = storedNotes.find(note => note.id === activeId)
+      headingDiv.value = heading
+      contentDiv.innerText = content
+    }
+  }
+}
+
+// initialize sidebar and set first note as active
+function init() {
+  const fragment = new DocumentFragment()
+  storedNotes.forEach((note, i) => {
+    fragment.appendChild(createNewSideNote(note))
+  })
+  sidebar.insertBefore(fragment, newButton)
+  makeNoteActive(storedNotes[0].id)
+}
+
+// change theme on toggle 
+toggle.addEventListener("change", () => {
+  if (toggle.checked) {
+    darkTheme()
+  } else {
+    lightTheme()
+  }
```

Comment:
As much as possible try and not use `else` statements. I feel the code is more clearer that way.
```suggestion
  if (toggle.checked) {
    darkTheme();
    return;
  }
  lightTheme();
```